deepchem/data/tests/test_setshard.py

```diff
@@ -0,0 +1,19 @@
+import deepchem as dc
+import numpy as np
+
+
+def test_setshard_with_X_y():
+  """Test setharding on a simple example"""
+  X = np.random.rand(10, 3)
+  y = np.random.rand(10,)
+  dataset = dc.data.DiskDataset.from_numpy(X, y)
+  assert dataset.get_shape()[0][0] == 10
```

Comment:
Maybe, 
```
X_shape, y_shape, _, _ == dataset.get_shape()
assert X_shape[0] == 10
assert y_shape[0] == 10
```
will be more meaningful. One can quickly understand what the values returned by `dataset.get_shape()` represents.